### DISCUSSION TITLE
Cannot run qvh

### DISCUSSION THREAD

**Discussion by Alex15190** [danielpaulus/quicktime_video_hack#135] [Q&A]
Hi. Could you please help me with this issue. I'm using M1 MacBook and I have this error
` ./qvh gstreamer
dyld[7535]: Library not loaded: '/usr/local/opt/libusb/lib/libusb-1.0.0.dylib'
  Referenced from: '/Users/check/Downloads/qvh'
  Reason: tried: '/usr/local/opt/libusb/lib/libusb-1.0.0.dylib' (no such file), '/usr/local/lib/libusb-1.0.0.dylib' (no such file), '/usr/lib/libusb-1.0.0.dylib' (no such file)
[1]    7535 abort      ./qvh gstreamer

brew install libusb
Warning: libusb 1.0.26 is already installed and up-to-date.
To reinstall 1.0.26, run:
  brew reinstall libusb`

**Comment by sauravbv** [comment]
Having the same issue..
